components/notebook-controller/main.go

```diff
@@ -68,7 +68,7 @@ func main() {
 	flag.IntVar(&Burst, "burst", 0, "If it's zero, the created RESTClient will use DefaultBurst")
 	flag.IntVar(&QPS, "qps", 0, "If it's zero, the created RESTClient will use DefaultQPS")
 	opts := zap.Options{
-		Development: true,
+		Development: false,
```

Comment:
I propose we keep the debug logs for notebook-controller, since those are useful during performance testing